modules/installation-vsphere-config-yaml.adoc

```diff
@@ -42,8 +42,6 @@ networking:
 ---
 platform:
   vsphere:
-    apiVIPs:
```

Comment:
As I'm removing these fields from the UPI docs, I should update the [Additional VMware vSphere configuration parameters](https://64295--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installation-config-parameters-vsphere#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere) table as it lists the `apiVIPs` and `ingressVIPs` YAML entries. Update them in regard to stating that they apply to the IPI `install-config.yaml` file ?

Reply:
@WenXinWei and @rvanderp3

Reply:
i agree. You can add a description to indicate that they are used for ipi.

Reply:
Thanks, @WenXinWei . Updated.